spec/features/webauthn/sign_in_spec.rb

```diff
@@ -15,13 +15,18 @@
     )
   end
 
-  it 'allows the user to sign in if webauthn is successful' do
-    mock_webauthn_verification_challenge
+  context 'with javascript enabled', :js do
```

Comment:
I restored _some_ JavaScript-enabled testing in this file, since part of the idea with #8761 was to have more realistic testing for how WebAuthn is expected to work in a real browser. I think it should suffice to do this once for the critical path, and use the default driver for everything else.